### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by swasthikcnayak**
added the quiz section, needs improvement in ui, uncomment in installed apps and djangoproject URLs in testing also in navbar if needed

### CODE REVIEW

File: quiz/locale/de/LC_MESSAGES/django.po [hunk 1]
```diff
@@ -0,0 +1,476 @@
+# SOME DESCRIPTIVE TITLE.
+# Copyright (C) YEAR THE PACKAGE'S COPYRIGHT HOLDER
+# This file is distributed under the same license as the PACKAGE package.
+# FIRST AUTHOR <EMAIL@ADDRESS>, YEAR.
```
Comment (18praneeth) [inline quiz/locale/de/LC_MESSAGES/django.po:4]: Are these files are necessary?

Comment (swasthikcnayak) [reply quiz/locale/de/LC_MESSAGES/django.po:4]: I think they are not.also remove the comments in urls, and installed apps after you try in your system and doing some modifications

Comment (18praneeth) [reply quiz/locale/de/LC_MESSAGES/django.po:4]: Sure

**Comment by 18praneeth** [comment]
image
can you updated the requirements.txt with the modules that you have used

**Review by 18praneeth — APPROVED** [review]

**Pull request merged**